src/backend/src/controllers/teams.controllers.ts

```diff
@@ -19,3 +20,46 @@ export const getSingleTeam = async (req: Request, res: Response) => {
 
   return res.status(200).json(teamTransformer(team));
 };
+
+export const editDescription = async (req: Request, res: Response) => {
+  const { body } = req;
+  const { userId, teamId, newDescription } = body;
+
+  const user = await prisma.user.findUnique({ where: { userId } });
+  const team = await prisma.team.findUnique({
+    where: { teamId },
+    include: {
+      leader: true,
+      projects: {
+        include: {
+          wbsElement: true
+        }
+      },
+      members: true
+    }
+  });
+  if (!user) return res.status(404).json({ message: `User with id #${userId} not found!` });
+  if (!team) return res.status(404).json({ message: `Team with id #${teamId} not found!` });
+
+  const canAccess = (user.role === Role.ADMIN || user.role === Role.APP_ADMIN) && userId === team?.leaderId;
```

Comment:
should be all `or` and also don't need `?` on team since we just checked if it was null